xapian-core/api/queryinternal.cc

```diff
@@ -783,7 +785,8 @@ Query::Internal::unserialise(const char ** p, const char * end,
 		    break;
 		default:
 		    // 8 to 12 are currently unused.
-		    throw SerialisationError("Unknown multi-way branch Query operator");
+		    throw SerialisationError(
+			    "Unknown multi-way branch Query operator");
```

Comment:
This line is over-length in the existing code, and fixing formatting problems is useful, but you aren't changing this part of the code so this isn't the PR to be addressing it in.  Mixing reformatting and functional changes just makes for a needlessly larger patch which is harder to review.